src/main/archetype/all/pom.xml

```diff
@@ -50,24 +50,36 @@
                 <extensions>true</extensions>
                 <configuration>
                     <group>${groupId}</group>
+                    <packageType>container</packageType>
                     <embeddeds>
                         <embedded>
                             <groupId>${groupId}</groupId>
                             <artifactId>${rootArtifactId}.ui.apps</artifactId>
                             <type>zip</type>
                             <target>/apps/${appId}-packages/application/install</target>
                         </embedded>
+                        <embedded>
+                            <groupId>${groupId}</groupId>
+                            <artifactId>${rootArtifactId}.core</artifactId>
+                            <target>/apps/${appId}-packages/application/install</target>
```

Comment:
The target folder ${appId}-packages doesn't seem like a good fit for bundles. Maybe one should rename to ${appId}-deployment-units or something like that. Also the intermediate level `application` seems superfluous to me.

Reply:
I agree with your concept.
Just one detail: `bundles` might be a name that is well established and better understood than `deployment-units`.
I'd therefore go for `<target>/apps/${appId}-bundles/install</target>`

Reply:
The [docs](https://docs.adobe.com/content/help/en/experience-manager-cloud-service/implementing/developing/aem-project-content-package-structure.html) are a bit unclear on this:

1. Text mentions initially: `/apps/my-app/install`
2. Diagram and details mention: `/apps/<app-name>-packages/(content|application)/install(.author|.publish)?`

I'd rather we stick to what's in the docs. If not, maybe update those as well.

Reply:
ok, let's then keep this as it is now…

Reply:
Please update the docs. The folder now contains both bundles and packages but the naming conventions are really only suitable for packages (both the package-type level content/application as well as the parent folder name)

Reply:
@kwin Can you please open a follow up issue for this? It should not be handled by this PR which is about a new version of the filevault maven plugin. Thanks!